src/installing-workflow/index.md

```diff
@@ -75,6 +75,15 @@ $ helmc install workflow-v2.7.0               # injects resources into
                                               # your cluster
 ```
 
+!!! Experimental
+	Workflow can also be installed now using the [Kubernetes Helm][helm]. All the details that are needed for a production deployments
+  like off-cluster storage, external registry etc., can be configured by passing an optional [values file][valuesfile] which overrides
+  default values.
+  ```
+  $ helm repo add deis https://charts.deis.com/workflow                                # add the workflow charts repo
+  $ helm install workflow --version=v2.7.0 --namespace=deis -f <optional values file>  # injects resources into your cluster
```

Comment:
@kmala I can't get `helm install workflow --version=v2.7.0 --namespace=deis` to work w/o changing it to `helm install deis/workflow ...` -- should the docs be updated to reflect this or am I missing something?